src/Stratis.Bitcoin.IntegrationTests/Miners/ProofOfStakeMintCoinsSteps.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Miner.Interfaces;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Controllers;
+using Stratis.Bitcoin.Features.Wallet.Models;
+using Stratis.Bitcoin.IntegrationTests.Builders;
+using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+using Xunit.Abstractions;
+using static Stratis.Bitcoin.Features.Miner.PosMinting;
+
+namespace Stratis.Bitcoin.IntegrationTests.Miners
+{
+    public partial class ProofOfStakeMintCoinsSpecification
+    {
+        private IDictionary<string, CoreNode> nodes;
+        private NodeGroupBuilder nodeGroupBuilder;
+        private const string PowMiner = "ProofOfWorkNode";
+        private const string PosStaker = "ProofOfStakeNode";
+        private SharedSteps sharedSteps;
+
+        private HdAddress powSenderAddress;
+        private Key powSenderPrivateKey;
+
+        private HdAddress posReceiverAddress;
+        private Key posReceiverPrivateKey;
+
+        private const string PowWallet = "powwallet";
+        private const string PowWalletAccount = "account 0";
+        private const string PowWalletPassword = "password";
+
+        private const string PosWallet = "poswallet";
+        private const string PosWalletAccount = "account 0";
+        private const string PosWalletPassword = "password";
+
```

Comment:
if the account and password are the same can remove two of the constants and share the other two.

Reply:
Done